CHANGELOG

```diff
@@ -1,14 +1,14 @@
 devel
 -----
 
+* Fixed BTS-418: Suboptimal index range calculation with redundant conditions.
+
 * Add missing metrics for user traffic: Histograms:
     `arangodb_client_user_connection_statistics_bytes_received`
     `arangodb_client_user_connection_statistics_bytes_sent`
   These numbers were so far only published via the statistics API.
   This is needed for Oasis traffic accounting.
 
-* Deleted customizable Pregel (AIR) and Greenspun library
```

Comment:
intentional change: this changelog entry appeared in the changelog twice before.